website/src/pages/auth/user-auth-layout.tsx

```diff
@@ -59,30 +56,19 @@ export const LoginHeaderButton = (props?: { className?: string }) => {
   )
 }
 
-const ConfirmLogout = (props?: { className?: string }) => {
+export const ConfirmLogout = (props?: { className?: string }) => {
   const { user } = useUser()
-  const popover = usePopoverState({ gutter: 2 })
 
   return (
-    <>
-      <PopoverDisclosure
-        {...popover}
-        className={cx(props?.className, cleanButton)}
-      >
-        {popover.visible ? "Cancel" : "Log out"}
-      </PopoverDisclosure>
-
-      <Popover {...popover} tabIndex={0}>
-        <Button
-          className={cx(props?.className, popoverButton)}
-          onClick={() => {
-            user?.signOut()
-          }}
-        >
-          Log out
-        </Button>
-      </Popover>
-    </>
+    <CleanButton
+      className={props?.className}
+      onClick={() => {
+        let confirmation = confirm("Are you sure you want to sign out?")
+        if (confirmation) user?.signOut()
```

Comment:
Flagging this to make sure we update after we merge the managed sign out function.